plugins/indentlist/plugin.js

```diff
@@ -174,7 +174,19 @@
 				listArray[ i ].indent += indentOffset;
 				// Make sure the newly created sublist get a brand-new element of the same type. (http://dev.ckeditor.com/ticket/5372)
 				if ( indentOffset > 0 ) {
-					var listRoot = listArray[ i ].parent;
+					var listRoot;
+
+					for ( var j = i - 1; j >= 0; j-- ) {
```

Comment:
Some comment with the issue reference could be useful here so it is clear what is the exact purpose of this loop.